R/support_links.R

```diff
@@ -0,0 +1,32 @@
+support_links <- function() {
+  div(
+    h2("Give us feedback"),
+    "This dashboard is a new service that we are developing. If you have any feedback or suggestions for improvements, please submit them using our ",
+    a(
+      href = "https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-c6JT6ONG3lJtlg-5hU4A6xURUpQME1OUVZIMEFMUUdNMEVONkhEN0g1VSQlQCN0PWcu",
```

Comment:
This needs updating to be a form for this specific dashboard, can create a form using microsoft forms with similar questions to the SCAP form - https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-fim2cxpeIFFus-loZuDRkhUN0ZPMlhCSDgzQjlVTjNVTFpXRTdWMlo0MC4u